addons/project/models/project.py

```diff
@@ -1633,6 +1633,8 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
             fields_list += [groupby] if isinstance(groupby, str) else groupby
         if domain:
             fields_list += [term[0].split('.')[0] for term in domain if isinstance(term, (tuple, list))]
+        # only take field name when having ':' e.g 'date_deadline:week' => 'date_deadline'
+        fields_list = [f.split(':')[0] for f in fields_list]
```

Comment:
Why not doing it only when we have some items in groupby parameter? 🤔 
I mean we could do it in line 1633 instead.

Reply:
it was my suggestion to combine the original 2 from 1633 and remove the one from 1631 (not done in current version). but indeed it would do it for field in domain when it's not necessary.

Reply:
I agree with your suggestion, it is good idea. However, your suggestion was not correctly applied in this PR when I reviewed it. 😬